packages/babel-plugin-polyfill-corejs2/src/index.ts

```diff
@@ -154,7 +157,7 @@ export default defineProvider<Options>(function (
           meta.key === "Symbol.iterator" &&
```

Comment:
(Line 141 above)

 We can cast `path.node` to `BinaryExpression`, thanks to the check at line 133. Also in the corejs3 plugin.

Reply:
Ah, I figured one of the conditionals would _implicitly_ narrow it, but did not know how to interpret that myself. Both have been cast now